docs/pages/brownfield/integration-with-existing-apps.mdx

```diff
@@ -0,0 +1,637 @@
+---
+title: Integrating a React Native view into existing native apps
+sidebar_title: Integrated approach
+description: Add Expo and React Native to existing native apps using the "integrated" approach.
+hideTOC: true
+---
+
+import { Terminal } from '~/ui/components/Snippet';
+import { Step } from '~/ui/components/Step';
+import { Tabs, Tab } from '~/ui/components/Tabs';
+
+React Native offers a flexible development strategy. Whether you're launching a new app or looking to integrate an existing native app, it provides a powerful solution. You can leverage its capabilities to introduce new features or views incrementally, allowing your team to adopt modern, cross-platform development without a complete rewrite.
+
+This guide will walk you through the steps to add a React Native view into an existing native app using the "integrated" approach.
+
+## Before you start
+
+To integrate React Native into your existing application, you'll need to set up a JavaScript development environment. This includes installing Node.js to run the Expo CLI and Yarn to manage the project's JavaScript dependencies.
+
+System requirements:
```

Comment:
```suggestion
```

We don't need this since the line above already explains the system requirements.